tools/setup/emoji/download-emoji-data

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+output_path=static/third/emoji-data
```

Comment:
Other places in the codebase seem to use capital letters for these types of variables